tests/flow/intersection/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -79,12 +79,14 @@ function hasObjectMode_ok(options: DuplexStreamOptions): boolean {
  * @flow
  */
 
-type DuplexStreamOptions = ReadableStreamOptions &
-  WritableStreamOptions & {
-    allowHalfOpen?: boolean,
-    readableObjectMode?: boolean,
-    writableObjectMode?: boolean
-  };
+type DuplexStreamOptions =
+  & ReadableStreamOptions
+  & WritableStreamOptions
+  & {
+      allowHalfOpen?: boolean,
+      readableObjectMode?: boolean,
+      writableObjectMode?: boolean
+    };
```

Comment:
This is the only one that is arguable, but in my experience it's rare that you combine two variables with an object